src/main/java/com/urswolfer/intellij/plugin/gerrit/ui/SettingsPanel.form

```diff
@@ -22,9 +22,6 @@
               <grid row="0" column="0" row-span="1" col-span="1" vsize-policy="0" hsize-policy="6" anchor="0" fill="1" indent="0" use-parent-layout="false"/>
             </constraints>
             <properties/>
-            <clientProperties>
```

Comment:
Please revert changes in this file which are not directly related to your change. I know, this is annoying - IntelliJ tries to cleanup things which are required (IMHO).